test-data/unit/sql-udf.test

```diff
@@ -75,7 +75,7 @@ pandas_udf(lambda *xs: 42, "str", PandasUDFType.GROUPED_AGG)
 
 
 [case mapIterUdf]
-from pyspark.sql.functions import pandas_udf, PandasUDFType
+from pyspark.sql.pandas.functions import pandas_udf, PandasUDFType
```

Comment:
I am also not so much aware of this code base; however, just wanted to note:

- original import will work as was.
- MAP ITER became `mapInPandas`

Reply:
> original import will work as was.

That's primarily to check both sides. In general MyPy is pretty strict about these things so

```
from pyspark.sql.pandas.functions import pandas_udf, PandasUDFType
```
would type check fine in `pyspark.sql.functions` but, not as transitive import. For that I had to switch to aliases.
```
from pyspark.sql.pandas.functions import pandas_udf as pandas_udf, PandasUDFType as PandasUDFType
```